reactor-core/src/main/java/reactor/core/publisher/FluxElapsed.java

```diff
@@ -46,6 +46,7 @@
 	@Override
 	public Object scanUnsafe(Attr key) {
 		if (key == Attr.RUN_ON) return scheduler;
+		if (key == Attr.RUN_STYLE) return Attr.RunStyle.ASYNC;
```

Comment:
as far as I see it is `SYNC`

Reply:
Fixed in commit [a69fc2a](https://github.com/reactor/reactor-core/pull/2123/commits/a69fc2ac7f60a06b480ab1273d97afaa47cfc036)